keps/sig-instrumentation/1013-metrics-watch-api/kep.yaml

```diff
@@ -0,0 +1,18 @@
+---
+title: Watch support for metrics APIs
```

Comment:
nit: can you please also set "kep-number" field?

[same below]

Reply:
Stupid question, but why do we still do the kep number thing?

Reply:
Makes validation a bit simpler (you don't have to parse the directory :) )